lib/matplotlib/axes/_base.py

```diff
@@ -1048,6 +1048,7 @@ def cla(self):
         self.tables = []
         self.artists = []
         self.images = []
+        self.child_axes = []
```

Comment:
stash them into self.artists?  I'd rather work towards unifying all these containers rather than adding more of them...

Reply:
Seec omment below for why not implimented....